include/deal.II/base/polynomials_barycentric.h

```diff
@@ -215,6 +215,31 @@ template <int dim>
 class BarycentricPolynomials : public ScalarPolynomialsBase<dim>
 {
 public:
+  /**
+   * Alias for polynomial type.
+   */
+  using PolyType = BarycentricPolynomial<dim>;
+
+  /**
+   * Alias for polynomial gradient type.
+   */
+  using GradType = std::array<PolyType, dim>;
+
+  /**
+   * Alias for polynomial hessian type.
+   */
+  using HessianType = std::array<GradType, dim>;
+
+  /**
+   * Alias for polynomial third derivatives type.
+   */
+  using ThirdDerivativesType = std::array<HessianType, dim>;
+
+  /**
+   * Alias for polynomial fourth derivatives type.
+   */
+  using FourthDerivativesType = std::vector<ThirdDerivativesType>;
```

Comment:
Did you mean to use
```suggestion
  using FourthDerivativesType = std::array<ThirdDerivativesType, dim>;
```
here as well?

Reply:
Indeed, it was just an oversight! Thanks for spotting it out :+1: